test/tests/01-package/run.sh

```diff
@@ -0,0 +1,30 @@
+# XXX why only ubuntu?
```

Comment:
mostly because I know it has `bash` for sure

Reply:
Changed it to `/bin/sh` and enabled for other distros